uw_sws/models.py

```diff
@@ -365,21 +365,21 @@ def int_key(self):
 
     def __eq__(self, other):
         return (other is not None and
-                type(self) == type(other) and
+                isinstance(other, Term) and
```

Comment:
TO fix style error:
E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`